make/test.mk

```diff
@@ -164,7 +175,7 @@ deploy-member:
 ifeq ($(MEMBER_REPO_PATH),)
 	$(eval MEMBER_REPO_PATH = /tmp/codeready-toolchain/member-operator)
 endif
-	oc new-project $(MEMBER_NS) 1>/dev/null
+	-oc new-project $(MEMBER_NS) 1>/dev/null
```

Comment:
I am not sure why are we using `-` here and also to redirect error to `dev/null`,error code is `2` not `1`

Reply:
Discussing here https://github.com/codeready-toolchain/toolchain-e2e/pull/12#discussion_r336439458